misc-scripts/xref_mapping/XrefParser/HGNCParser.pm

```diff
@@ -456,6 +457,8 @@ sub add_synonyms_for_hgnc {
     $dead_string =~ s/"//xg;
     my @dead_array = split( ',\s', $dead_string );
     foreach my $dead (@dead_array){
+      utf8::decode($dead);
```

Comment:
I recommend using Encode::decode. Encode is a core module. While utf8::decode works for the greek characters we have here, Encode::decode is more robust and can deal with invalid data. Since we don't have any control over the data we get, I think Encode::decode is safer.
I'd also drop the "use utf8;" - this is only useful if you use utf8 characters in the source code, which does not seem to be the case here.
Example usage:
```
use Encode;
my $str = decode("UTF-8", $bytes);
```

Reply:
Hi Arne. Thanks for the review. While I agree with you, I only used utf8::decode because utf8::encode in used earlier in the script, so I kept it for consistency. I will however test around with you suggestions and update accordingly.